src/include/migraphx/algorithm.hpp

```diff
@@ -90,6 +90,27 @@ levenshtein_distance(Iterator1 first1, Iterator1 last1, Iterator2 first2, Iterat
     return std::ptrdiff_t{1} + std::min({x1, x2, x3});
 }
 
+inline size_t levenshtein_distance(const std::string& s1, const std::string& s2)
```

Comment:
I don't see the previous implementation being used anywhere. Maybe we should remove it.

Reply:
OK. Agreed. May I move this new algorithm to the `src/driver` subdirectory as well? Thanks.

Reply:
It was originally designed to be here because of possible reuse in other situations. Can't think of any right now, but probably not worth moving just yet.